### PR TITLE
Add management fabric integration as a dev-preview

### PR DESCRIPTION
**Pull request by clyang82**
ref: https://issues.redhat.com/browse/ACM-14141

### CODE REVIEW

File: dev-preview/README.md [hunk 1]
```diff
@@ -199,3 +200,172 @@ The OpenShift Cluster API Operator deploys the main components (CAPI & CAPA depl
...
+Management fabric is designed to establish a common hosted integration fabric and control plane which can be plugged into each Red Hat management product or service, as well as consumed by our ISV partners.
+
+### Bring Management Fabric via the global hub operator
+
+The multicluster global hub can be able to install Management fabric in the on-premise environment.
+
+#### Install the global hub operator
+
+Refer to [here](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html/multicluster_global_hub/multicluster-global-hub#global-hub-install-connected) to install the multicluster global hub in your environment.
+
```
Comment (bjoydeep) [inline dev-preview/README.md:215]: this document says we need a ACM Hub as a pre-req. We should clarify that.

File: dev-preview/README.md [hunk 2]
```diff
@@ -199,3 +200,172 @@ The OpenShift Cluster API Operator deploys the main components (CAPI & CAPA depl
 ### Search-v2 - Odyssey
 
   * Search-v2 brings a re-architected backbone facilitating greater scale and resiliance within the service.
+
+## Global Hub Integration with Management Fabric
+
+Management fabric is designed to establish a common hosted integration fabric and control plane which can be plugged into each Red Hat management product or service, as well as consumed by our ISV partners.
+
```
Comment (bjoydeep) [inline dev-preview/README.md:207]: A management service creates, modifies or deletes a managed resource in its local inventory. To aid in the integration of the Red Hat Hybrid Cloud Management solution, they will report all of these operations to the management fabric via the Asset Inventory APIs. With the Asset Inventory APIs, these services are referenced as Reporters.

What about this:

To aid the integration of the Red Hat Hybrid Cloud Management solutions, Management Fabric was created. A management solution like ACM creates, modifies or deletes a managed resource in its local inventory. Also, all these operations will be reported to the management fabric via the Asset Inventory APIs. Other Red Hat Hybrid Cloud Management solutions will follow. Customers and ISV partners will be able to plug into Management Fabric to subscribe to these change events.

File: dev-preview/README.md [hunk 3]
```diff
@@ -199,3 +200,172 @@ The OpenShift Cluster API Operator deploys the main components (CAPI & CAPA depl
...
+#### Create the global hub instance
+
+You need to add a new annotation `global-hub.open-cluster-management.io/with-inventory: ""` in the global hub instance to enable the inventory-api deployment in the global hub namespace.
+You should be able to see the inventory-api deployment in the global hub namespace. The result likes:
+```
+inventory-api-6c7567fcfb-n5qv9                  	                       1/1 	Running   0         	11h
+inventory-api-f33567fcfb-p9zj5                  	                       1/1 	Running   0           11h
+```
+If you install the multicluster global hub with customized postgresql, you need to manually create `inventory` database.
+
```
Comment (bjoydeep) [inline dev-preview/README.md:225]: May be should remove this to keep this simple. wdyt?

File: dev-preview/README.md [hunk 4]
```diff
@@ -199,3 +200,172 @@ The OpenShift Cluster API Operator deploys the main components (CAPI & CAPA depl
...
+# access pod to check topics
+oc exec -it kafka-kafka-0 sh
+bin/kafka-console-consumer.sh --bootstrap-server $kafka_bootstrap_server:443 --consumer.config=/tmp/client.properties --topic kessel-inventory --from-beginning
+```
+The result looks like this:
+```
+{"metadata":{"id":0,"last_reported":"2024-09-11T15:45:08.941075631Z","resource_type":"k8s-cluster","workspace":"","labels":null},"reporter_data":{"reporter_instance_id":"guest","reporter_type":"ACM","last_reported":"2024-09-11T15:45:08.941075631Z","local_resource_id":"1","reporter_version":"0.1","console_href":"www.example.com","api_href":"www.example.com"},"resource_data":{"external_cluster_id":"1234","cluster_status":"READY","kube_version":"1.31","kube_vendor":"OPENSHIFT","vendor_version":"4.16","cloud_platform":"AWS_UPI","nodes":[{"name":"www.example.com","cpu":"7500m","memory":"30973224Ki","labels":[{"key":"has_monster_gpu","value":"yes"}]}]}}
+```
+Congratulations! You have set up the environment correctly.
+
```
Comment (bjoydeep) [inline dev-preview/README.md:304]: A Red Hat management tool like ACM would have automatically registered its cluster and policies etc using this API. In that case, you would directly see the events without having to make the API POST call.

File: dev-preview/README.md [hunk 5]
```diff
@@ -199,3 +200,172 @@ The OpenShift Cluster API Operator deploys the main components (CAPI & CAPA depl
...
+            "reporter_version": "2.11.3"
+        },
+        "relationship_data": {
+            "status": "VIOLATIONS",
+            "k8s_policy_id": 0,
+            "k8s_cluster_id": 0
+        }
+    }
+}
+```
```
Comment (bjoydeep) [inline dev-preview/README.md:371]: Why not also add the same message :)
Congratulations! You have set up the environment correctly.

**Review by bjoydeep — APPROVED** [review]
/lgtm

**Comment by bjoydeep** [comment]
@braddmin this looks good. Would you mind taking a quick look and approving it please

**Comment by braddmin** [comment]
> @braddmin this looks good. Would you mind taking a quick look and approving it please

Approved

**Review by jnpacker — APPROVED** [review]

**Pull request merged**